lib/assets/DefaultFastfileTemplate

```diff
@@ -24,32 +24,11 @@ default_platform :ios
 platform :ios do
   before_all do
     # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
-    cocoapods
-    carthage
-    # increment_build_number
-
-    xctool # run the tests of your app
-  end
-
-  desc "Runs all the tests"
-  lane :test do
-    snapshot
-  end
-
-  desc "Submit a new Beta Build to Apple TestFlight"
-  desc "This will also make sure the profile is up to date"
-  lane :beta do
```

Comment:
Per some conversation above, I had removed the original FastfileTemplate and replaced it with this DefaultFastfileTemplate. It looks like i deleted the lanes here, but based on your suggestion, I had deleted the file, so git is noticing it as a rename.